interface/main/messages/js/reminder_appts.js

```diff
@@ -0,0 +1,464 @@
+/**
+ * /interface/main/messages/js/reminder_appts.js
+ *
+ * Copyright (C) 2017 MedEx <support@MedExBank.com>
+ *
+ * LICENSE: This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package LibreHealth EHR
+ * @author MedEx <support@MedExBank.com>
+ * @link http:Librehealth.io
+ */
+
+
+var labels=[];
+var postcards=[];
+var show_just;
+
+$(function() {
+  $( "#datepicker1" ).datepicker({ dateFormat: "yy-mm-dd" });
```

Comment:
no it is not the same, also the one in patient portal is the same they both will need to be changes at some point in time